app/src/main/java/com/sun/music_66/extentions/ViewExtension.kt

```diff
@@ -0,0 +1,10 @@
+package com.sun.music_66.extentions
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.annotation.LayoutRes
+
+fun ViewGroup?.inflate(inflater: LayoutInflater, @LayoutRes layoutResId: Int): View {
```

Comment:
```suggestion
fun ViewGroup.inflate(@LayoutRes layoutResource: Int)  = LayoutInflater.from(context).inflate(layoutResource, this, false)
```

Nếu em muốn viết function này ngắn gọn hơn
có thể nên viết theo cách này